posts/2019-04-25-Rust-1.34.1.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Announcing Rust 1.34.1"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.34.1, and a new version of rustup, 1.18.1.
+Rust is a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via rustup,
+getting Rust 1.34.1 and rustup 1.18.1 is as easy as:
+
+```console
+$ rustup update stable
```

Comment:
I don't think this updates rustup, for that you need to run without arguments.

Maybe add another line with `rustup self update`?

Reply:
I thought the same but @pietroalbini checked this.

Reply:
Rustup automatically self-updates on any update unless `--no-self-update` is passed :D  (Feature as of 1.17.0)

Reply:
Ah I see, the rustup 1.17.0 CLI help doesn't make this very clear